grafana/RELEASES.md

```diff
@@ -0,0 +1,6 @@
+# Release Notes for Grafana container.
+
+## 2.1.0 (9-28-2015)
```

Comment:
My suggestion is to keep the Grafana version as 2.1.0 and tag on the heapster version. So for 1.1 release it would be 2.1.0-v4 and master 2.1.0-v5. 

What do you think?